annotate.el

```diff
@@ -2084,22 +2084,29 @@ from a chain where `annotation' belong."
 
 (defun annotate--delete-annotation-chain-prevent-modification (annotation)
 "Delete an annotation chain backing up and restoring modification
-status of the buffer before deltion occured."
+status of the buffer before deletion occured.
+
+This function is not part of the public API."
   (annotate-ensure-annotation (annotation)
     (annotate-with-restore-modified-bit
      (annotate--delete-annotation-chain annotation))))
 
+(defun annotate--confirm-annotation-delete ()
```

Comment:
I think we should use `yes-or-no-p` instead of implementing the prompt ourselves. That way the form of the prompt is standardized. For example, I like to alias my `yes-or-no-p` to `y-or-n-p`, to save me some typing.

Reply:
Hi!

I just did not know about this function, sure is the best solution instead of a custom one! (and also i learnt something new today)!

Bye!
C.